plugins/suspendmanager.cpp

```diff
@@ -698,11 +714,14 @@ DFhackCExport command_result plugin_load_site_data (color_ostream &out) {
         DEBUG(control,out).print("no config found in this save; initializing\n");
         config = World::AddPersistentSiteData(CONFIG_KEY);
         config.set_bool(CONFIG_IS_ENABLED, is_enabled);
+        config.set_bool(CONFIG_PREVENT_BLOCKING, suspendmanager_instance->prevent_blocking);
```

Comment:
this should initialize the new config data to a default, not whatever happens to be in the suspendmanager_instance from the previously loaded fort